src/Wallabag/CoreBundle/Resources/translations/messages.it.yml

```diff
@@ -334,41 +334,41 @@ import:
-    welcome_message: 'Welcome to the wallabag API'
-    documentation: 'Documentation'
-    how_to_first_app: 'How to create my first application'
-    full_documentation: 'View full API documentation'
-    list_methods: 'List API methods'
+    welcome_message: 'Benvenuto nelle API di wallabag'
+    documentation: 'Documentazione'
+    how_to_first_app: 'Come creare la mia prima applicazione'
+    full_documentation: 'Consulta la documentazione API completa'
+    list_methods: 'Elenco dei metodi API'
     clients:
-        title: 'Clients'
-        create_new: 'Create a new client'
+        title: 'Client'
+        create_new: 'Crea un nuovo client'
     existing_clients:
-        title: 'Existing clients'
+        title: 'Client esistenti'
         field_id: 'Client ID'
         field_secret: 'Client secret'
-        field_uris: 'Redirect URIs'
-        field_grant_types: 'Grant type allowed'
-        no_client: 'No client yet.'
+        field_uris: 'Redirect URI'
+        field_grant_types: 'Tipi di grant permessi'
+        no_client: 'Ancora nessun client.'
     remove:
-        warn_message_1: 'You have the ability to remove this client. This action is IRREVERSIBLE !'
-        warn_message_2: "If you remove it, every app configured with that client won't be able to auth on your wallabag."
-        action: 'Remove this client'
+        warn_message_1: "Hai la possibilitò di rimuovere questo client.L'operazione è IRREVERSIBILE!"
+        warn_message_2: "Se lo rimuovi, ogni app configurata con questo client non sarà più in grado di autenticarsi."
+        action: 'Rimuovi questo client'
     client:
-        page_title: 'Developer > New client'
-        page_description: 'You are about to create a new client. Please fill the field below for the redirect URI of your application.'
+        page_title: 'Sviluppatori > Nuovo client'
+        page_description: 'Stai per creare un nuovo client. Compila i campi sottostanti per il redirect URI della tua applicazione.'
         form:
-            redirect_uris_label: 'Redirect URIs'
-            save_label: 'Create a new client'
-        action_back: 'Back'
+            redirect_uris_label: 'Redirect URI'
+            save_label: 'Crea un nuovo client'
+        action_back: 'Indietro'
     client_parameter:
-        page_title: 'Developer > Client parameters'
-        page_description: 'Here are your client parameters.'
+        page_title: 'Sviluppatori > parametri Client'
+        page_description: 'Questi sono i tuoi parametri del client.'
         field_id: 'Client ID'
         field_secret: 'Client secret'
-        back: 'Back'
-        read_howto: 'Read the howto "Create my first application"'
+        back: 'Indietro'
+        read_howto: 'Leggi howto "Come creare la mia prima applicazione"'
     howto:
-        page_title: 'Developer > How to create my first application'
+        page_title: 'Sviluppatori > Come creare la mia prima applicazione'
         description:
```

Comment:
all this block is not translated. (no need to open a new pull request for that). Create a new commit, it'll be added in this pull request).